### PR TITLE
WIP: Implement commands as magics.

### PR DESCRIPTION
**Pull request by danielballan**
Needs tests and also %log_pos

### CODE REVIEW

File: ophyd/magics.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+from IPython.core.magic import Magics, magics_class, line_magic
+from operator import attrgetter
+
+@magics_class
+class SPECMagics(Magics):
+
+    positioners = []
+
+    @line_magic
+    def wa(self, line):
```
Comment (stuartcampbell) [inline ophyd/magics.py]: Is it worth adding something saying what `wa` does for the uninitiated? 😄

Comment (danielballan) [inline ophyd/magics.py]: Good suggestion. Done and force-pushed.

**Comment by danielballan** [comment]
I found a nice way to implement `%wa` without an ophyd dependency, so I'm putting this in `bluesky.magics` for now (in https://github.com/NSLS-II/bluesky/pull/720). Separately we can discuss whether we want to pull that out as a separate repo, but it does seem nice to have all the magics in one module -- wherever it lives.

**Comment by danielballan** [comment]
Specifically, it's here: https://github.com/NSLS-II/bluesky/pull/720/commits/3cab61fa830d6e7d3c8ee1c041f010aafd43d2d0#diff-9cbbeaee4ee19a2f0f7fd6cae424b824

**Pull request closed without merging**